test/analytics.test.js

```diff
@@ -495,7 +495,7 @@ describe('Analytics', function() {
     });
 
     it('should emit "invoke" with facade', function(done) {
-      var opts = { All: false };
+      var opts = { All: true };
```

Comment:
can we revert this change. I wonder if we need this change even after fixing the `emitter` issue.

Reply:
resolved by https://github.com/segmentio/analytics.js-core/pull/150/commits/fccfece9c33838866a52cb792e85cc3f3060d51b